mojave/code/modules/mob/living/basic/hostile_animals.dm

```diff
@@ -39,6 +39,19 @@
 /datum/ai_behavior/basic_melee_attack/ms13/hostile_animal
 	action_cooldown = 1.5 SECONDS
 
+//Some simple sidestepping; mainly for robots
+/datum/ai_behavior/basic_melee_attack/ms13/hostile_animal/sidestep
```

Comment:
This is gross and should just be applied as a robot behavior subtype.

Reply:
Sidestepping could still be a generic behavior of a melee attacking mob but I'll retype this into robots